specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-11-01/streamingservice.json

```diff
@@ -0,0 +1,2877 @@
+        },
+        "name": {
+          "type": "string",
+          "description": "Operation Id of the async operation."
+        },
+        "status": {
+          "description": "Operation status of the async operation.",
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "InProgress"
+          ],
+          "x-ms-enum": {
+            "name": "AsyncOperationStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "The status of an async operation.",
+      "type": "object"
+    },
+    "IngestInterruption": {
+      "properties": {
+        "begin": {
+          "description": "UTC time of interruption start, encoder disconnected.",
+          "type": "string"
+        },
+        "end": {
+          "description": "UTC time of interruption end, encoder re-connected.",
+          "type": "string"
+        },
+        "durationInSec": {
+          "description": "Duration of interruption in seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "reason": {
+          "description": "Interruption reason.",
+          "type": "string"
+        }
+      },
+      "description": "The live event ingest interruption data.",
+      "type": "object"
+    },
+    "Ingestion": {
+      "properties": {
+        "streamName": {
+          "description": "Ingestion stream name.",
+          "type": "string"
+        },
+        "beginTime": {
+          "description": "Ingestion begin time in UTC.",
+          "type": "string"
+        },
+        "endTime": {
+          "description": "Ingestion end time in UTC. Empty if it's not stopped yet.",
+          "type": "string"
+        },
+        "endReason": {
+          "description": "End reason.",
+          "type": "string"
+        },
+        "ingestInterruptions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IngestInterruption"
+          },
+          "x-ms-identifiers": [],
+          "description": "IngestInterruption entry list."
+        }
+      },
+      "description": "The live event ingestion telemetry data.",
+      "type": "object"
+    },
+    "TrackStatus": {
+      "properties": {
+        "trackId": {
+          "description": "Track Id.",
+          "type": "string"
+        },
+        "expectedBitrate": {
+          "description": "Expected bitrate for this track.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "incomingBitrate": {
+          "description": "Average incoming bitrate for last 20 seconds when live event is running.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "ingestDrift": {
+          "description": "Current ingest drift value in seconds for last 1 min.",
+          "type": "string"
+        }
+      },
+      "description": "The live event track status.",
+      "type": "object"
+    },
+    "Status": {
```

Comment:
I think "Status" is too generic (this will get mixed with all the other types in the SDK client libraries). Perhaps "LiveEventStatus"?